### PR TITLE
feat: add attest plugin that publish SBOM attestations for blob

### LINKED ISSUE #1: Initial setup
1. Write a tool publishing SBOM attestations
1. Set up the release workflow
    - Use GoReleaser
    - cf. https://github.com/aquasecurity/trivy/blob/main/goreleaser-canary.yml
1. Set up `plugin.yaml`
    - See https://aquasecurity.github.io/trivy/v0.32/docs/advanced/plugins/

### PR DESCRIPTION
**Pull request by otms61**
## Description

I have added attest plugin that publish SBOM attestations for blob.

I have confirmed that the following commands work.
```
> trivy plugin --debug install github.com/otms61/trivy-plugin-attest
2022-10-13T03:54:14.799+0900	INFO	Installing the plugin from github.com/otms61/trivy-plugin-attest...
2022-10-13T03:54:19.866+0900	INFO	Loading the plugin metadata...
2022-10-13T03:54:19.867+0900	DEBUG	Installing the plugin to /Users/saso/.trivy/plugins/attest...
2022-10-13T03:54:19.867+0900	DEBUG	Platform found, os: darwin, arch: amd64
2022-10-13T03:54:19.867+0900	DEBUG	Downloading the execution file from https://github.com/otms61/trivy-plugin-attest/releases/download/v0.0.5/trivy_plugin_attest_0.0.5_macOS-64bit.tar.gz...

> trivy attest cyclonedx ./sbom.cdx.json target/debug/bat
```

After it is released in the aquasecurity repository, we can install this plugin with the following command.
```
> trivy plugin install github.com/aquasecurity/trivy-plugin-attest
```

## Related issues

- Close https://github.com/aquasecurity/trivy-plugin-attest/issues/1

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+          name: trivy_plugin_attest_macOS-64bit
+          path: dist/trivy_plugin_attest_*_macOS-64bit.tar.gz
+          if-no-files-found: error
+
+      - name: Upload artifacts (trivy_plugin_attest_macOS-ARM64)
+        uses: actions/upload-artifact@v3
+        with:
+          name: trivy_plugin_attest_macOS-ARM64
+          path: dist/trivy_plugin_attest_*_macOS-ARM64.tar.gz
+          if-no-files-found: error
```
Comment (knqyf263) [inline .github/workflows/release.yaml]: We don't need to upload artifacts.

File: goreleaser.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      - -X main.version={{.Version}}
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - darwin
+      - linux
+    goarch:
+      - amd64
+      - arm64
+
```
Comment (knqyf263) [inline goreleaser.yml:18]: We should have `release` here.
https://goreleaser.com/customization/release/

File: .github/workflows/release.yaml (hunk 1 above)
Comment (otms61) [reply .github/workflows/release.yaml]: done

File: goreleaser.yml (hunk 2 above)
Comment (otms61) [reply goreleaser.yml:18]: We don't have to add a release section because the default release settings will upload the artifact to the release page.

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**